atmo/coordinator/coordinator_request.go

```diff
@@ -0,0 +1,81 @@
+package coordinator
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/pkg/errors"
+	"github.com/suborbital/atmo/directive"
+	"github.com/suborbital/reactr/request"
+	"github.com/suborbital/vektor/vk"
+)
+
+func (c *Coordinator) vkHandlerForDirectiveHandler(handler directive.Handler) vk.HandlerFunc {
+	return func(r *http.Request, ctx *vk.Ctx) (interface{}, error) {
+		req, err := request.FromVKRequest(r, ctx)
+		if err != nil {
+			ctx.Log.Error(errors.Wrap(err, "failed to request.FromVKRequest"))
+			return nil, vk.E(http.StatusInternalServerError, "failed to handle request")
+		}
+
+		// this should probably be factored out into the CoordinateRequest object
```

Comment:
is this the right issue? https://github.com/suborbital/reactr/issues/105

Reply:
@jagger27 Yes, that's right